addons/binding/org.openhab.binding.robonect/README.md

```diff
@@ -0,0 +1,132 @@
+# Robonect Binding
+
+Robonect is a piece of hardware which has to be put into your Husqvarna, Gardena and other branded automower and makes 
+it accessable in your internal network. More details about the Robonect module can be found at [robonect.de](http://www.robonect.de)
+
+This binding integrates mowers having the robonect module installed as thing into the openhab installation, allowing to
```

Comment:
ok. So far I plan to contribute it to the openhab addons.... however, who knows :-)